core-java-modules/core-java-sockets/src/main/java/com/baeldung/threading/request/RequestHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.threading.request;
+
+import java.io.PrintWriter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestHandler extends Thread {
```

Comment:
For clarity let's rename:
```suggestion
public class ThreadPerRequest extends Thread {
```